resharper/src/CSharp/Completions/InputActionMethods.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace JetBrains.ReSharper.Plugins.Godot.CSharp.Completions
+{
+    internal static class InputActionMethods
+    {
+        internal static readonly List<string> Methods = new()
+        {
+            "ActionPress", "ActionRelease", "GetActionRawStrength", "GetActionStrength", "GetAxis",
+            "IsActionJustPressed", "IsActionJustReleased", "IsActionPressed"
```

Comment:
Can we drop the check over hard-coded list of methods? And check for the `StringName` type of the parameter instead?

Reply:
Sounds good. I will take a look at that in the evening 👍

Reply:
Okay... I tried my luck for a few hours. But I'm can't wrap my head around traversing the syntax tree. But I think it should be doable. We could even check for the type of `ILocalVariableDeclaration` and if it is a `NodePath` we could support auto complete for something like this:

```CSharp
NodePath test = "Icon"
```

Could give me a hint of how to implement that?

Reply:
I had to do some trial-error, please see https://github.com/JetBrains/godot-support/pull/102/commits/2a663deb8ae7b3d1db8dcd4840c610afb740841c

Reply:
Would you do the same for `LookupNodePaths`?

Reply:
Uff easier than I thought! I added support for `NodePath` variable declarations as well.